placement-rules-in-sql.md

```diff
@@ -5,276 +5,363 @@ summary: Learn how to schedule placement of tables and partitions using SQL stat
-```
+## Limitations
 
-Then attach the policy to a table or partition using either `CREATE TABLE` or `ALTER TABLE`. Then, the placement rules are specified on the table or the partition:
+- To simplify maintenance, it is recommended to limit the number of placement policies within a cluster to 10 or fewer.
+- It is recommended to limit the total number of tables and partitions attached with placement policies to 10,000 or fewer. Attaching policies to too many tables and partitions can increase computation workloads on PD, thereby affecting service performance.
+- It is recommended to use the Placement Rules in SQL feature according to examples provided in this document rather than using other complex placement policies.
 
-```sql
-CREATE TABLE t1 (a INT) PLACEMENT POLICY=myplacementpolicy;
-CREATE TABLE t2 (a INT);
-ALTER TABLE t2 PLACEMENT POLICY=myplacementpolicy;
-```
+## Prerequisites
 
-A placement policy is not associated with any database schema and has the global scope. Therefore, assigning a placement policy does not require any additional privileges over the `CREATE TABLE` privilege.
+Placement policies rely on the configuration of labels on TiKV nodes. For example, the `PRIMARY_REGION` placement option relies on the `region` label in TiKV.
 
-To modify a placement policy, you can use [`ALTER PLACEMENT POLICY`](/sql-statements/sql-statement-alter-placement-policy.md), and the changes will propagate to all objects assigned with the corresponding policy.
+<CustomContent platform="tidb">
 
-```sql
-ALTER PLACEMENT POLICY myplacementpolicy FOLLOWERS=5;
+When you create a placement policy, TiDB does not check whether the labels specified in the policy exist. Instead, TiDB performs the check when you attach the policy. Therefore, before attaching a placement policy, make sure that each TiKV node is configured with correct labels. The configuration method for a TiDB Self-Hosted cluster is as follows:
+
+```
+tikv-server --labels region=<region>,zone=<zone>,host=<host>
 ```
 
-To drop policies that are not attached to any table or partition, you can use [`DROP PLACEMENT POLICY`](/sql-statements/sql-statement-drop-placement-policy.md):
+For detailed configuration methods, see the following examples:
 
-```sql
-DROP PLACEMENT POLICY myplacementpolicy;
-```
+| Deployment method | Example |
+| --- | --- |
+| Manual deployment | [Schedule replicas by topology labels](/schedule-replicas-by-topology-labels.md) |
+| Deployment with TiUP | [Geo-distributed deployment topology](/geo-distributed-deployment-topology.md) |
+| Deployment with TiDB Operator | [Configure a TiDB cluster in Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/configure-a-tidb-cluster#high-data-high-availability) |
 
-## View current placement rules
+> **Note:**
+>
+> For TiDB Dedicated clusters, you can skip these label configuration steps because the lables on TiKV nodes in TiDB Dedicated clusters are configured automatically.
```

Comment:
```suggestion
> For TiDB Dedicated clusters, you can skip these label configuration steps because the label on TiKV nodes in TiDB Dedicated clusters are configured automatically.
```